src/logs/metadata_engine/me_id.py

```diff
@@ -19,19 +19,45 @@
 int64 = ctypes.c_int64
 
 
-def meid_md5(entity_type: str, hashing_input: str):
-    long_id = _legacy_entity_id_md5(hashing_input)
+def meid_md5(entity_type: str, *hashing_input: str):
+    if None in hashing_input:
+        # any None value in input is treated as missing required data, no ID can be calculated
+        return None
+
+    concatented_input = "".join(hashing_input)
+
+    long_id = _legacy_entity_id_md5(concatented_input)
+    identifier = _encode_me_identifier(entity_type, long_id)
+    return identifier
+
+
+def meid_murmurhash(entity_type: str, *hashing_input: str) -> str:
+    if None in hashing_input:
+        # any None value in input is treated as missing required data, no ID can be calculated
+        return None
+
+    concatented_input = "".join(hashing_input)
+
+    long_id = _murmurhash2_64A(concatented_input)
     identifier = _encode_me_identifier(entity_type, long_id)
     return identifier
 
+def meid_murmurhash_awsseed(entity_type: str, *hashing_input: str) -> str:
+    if None in hashing_input:
+        # any None value in input is treated as missing required data, no ID can be calculated
+        return None
+
+    concatented_input = "".join(hashing_input)
+
+    long_id = _murmurhash2_64A(concatented_input, seed = -512093083)
```

Comment:
Can we have hex here so it is easier to find corresponding thing on cluster side? I mean for the seed

Reply:
At least put that hex from java in a comment, for reference

Reply:
Yes, that's what I did + a jira for reference
https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/pull/15/commits/4af3dc0188dc1f257d8f5fbf636daa10069152bb